mysql-plugin/src/main/java/co/cask/mysql/MysqlAction.java

```diff
@@ -48,7 +49,8 @@ public MysqlAction(MysqlActionConfig mysqlActionConfig) {
 
     @Name(MysqlConstants.AUTO_RECONNECT)
     @Description("Should the driver try to re-establish stale and/or dead connections")
-    public Boolean autoReconnect;
+    @Nullable
+    public Boolean autoReconnect = false;
```

Comment:
Why not use primitive type which is by default false? And this way you don't need to use nullable. Also please make sure you have default value set in the widget.

Reply:
done